### PR TITLE
theme: add blue-owl

### PR DESCRIPTION
**Pull request by alchatti**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes/features)

### Description

Created a new theme called `blue-owl`
Created file `blue-owl.omp.json` in `oh-my-posh/themes/`

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: themes/blue-owl.omp.json [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+            "home_icon": "\uF7DB",
+            "folder_icon": "",
+            "folder_separator_icon": "<transparent> \ue0bd </>",
+            "style": "agnoster_short"
+          }
+        },
+        {
+          "type": "git",
+          "style": "powerline",
+          "powerline_symbol": "",
```
Comment (JanDeDobbeleer) [inline themes/blue-owl.omp.json]: Can you use the `\u` representation of icons? Make the search easier for people looking to replace/tweak. Thanks!

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: themes/blue-owl.omp.json (hunk 1 above)
Comment (alchatti) [reply themes/blue-owl.omp.json]: Thanks, I have pushed an update with the \u representation of icons for Nerd Fonts icons.

**Comment by alchatti** [comment]
@JanDeDobbeleer  Thanks, I have pushed an update with the \u representation of icons for Nerd Fonts icons.

File: themes/blue-owl.omp.json (hunk 1 above)
Comment (alchatti) [reply themes/blue-owl.omp.json]: Thanks, I have pushed an update with the \u representation of icons for Nerd Fonts icons.

File: themes/blue-owl.omp.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        {
+          "type": "path",
+          "style": "diamond",
+          "leading_diamond": "<transparent,#01579B>\uE0B0</>",
+          "trailing_diamond": "\uE0B0",
+          "foreground": "#ffffff",
+          "background": "#01579B",
+          "properties": {
+            "home_icon": "\uF7DB",
+            "folder_icon": "",
```
Comment (JanDeDobbeleer) [inline themes/blue-owl.omp.json]: Missed a few 😉

File: themes/blue-owl.omp.json [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+      "type": "prompt",
+      "alignment": "left",
+      "segments": [
+        {
+          "type": "root",
+          "style": "plain",
+          "foreground": "#FFEB3B",
+          "background": "#a313a8",
+          "properties": {
+            "root_icon": "⚡"
```
Comment (JanDeDobbeleer) [inline themes/blue-owl.omp.json:19]: Missed a few 😉

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: themes/blue-owl.omp.json (hunk 3 above)
Comment (alchatti) [reply themes/blue-owl.omp.json:19]: That is an emoji, https://emojipedia.org/high-voltage/ any recommendation on how to proceed is much appreciated.
😁

Comment (alchatti) [reply themes/blue-owl.omp.json:19]: Fixed the folder_icon to \u representation, hope that covers it all.

File: themes/blue-owl.omp.json (hunk 2 above)
Comment (alchatti) [reply themes/blue-owl.omp.json]: Thanks, fixed the folder_icon to \u representation.

File: themes/blue-owl.omp.json (hunk 3 above)
Comment (JanDeDobbeleer) [reply themes/blue-owl.omp.json:19]: Let's leave it!

**Review by JanDeDobbeleer — APPROVED** [review]

**Comment by JanDeDobbeleer** [comment]
@All-contributors please add @alchatti for design

**Pull request merged**